test/tools/CodeCoverageAutomation/Start-CodeCoverageRun.ps1

```diff
@@ -4,6 +4,97 @@
     [Parameter(Position = 2)] $azureLogDrive = "L:\"
 )
 
+# Read the XML and create a dictionary for FileUID -> file full path.
+function GetFileTable()
+{
+    $files = $script:covData | Select-Xml './/File'
+    foreach($file in $files)
+    {
+        $script:fileTable[$file.Node.uid] = $file.Node.fullPath
+    }
+}
+
+# Get sequence points for a particular file
+function GetSequencePointsForFile([string] $fileId)
+{
+    $lineCoverage = @{}
+    $sequencePoints = $script:covData | Select-Xml ".//SequencePoint[@fileid = `"$fileId`"]"
+
```

Comment:
does this work with single quotes? If so, then:
`".//SequencePoint[@fileid='$fileid']"`
should work